app/api/services/brief_responses.py

```diff
@@ -10,15 +10,20 @@ def __init__(self, *args, **kwargs):
         super(BriefResponsesService, self).__init__(*args, **kwargs)
 
     def get_brief_responses(self, brief_id, supplier_code):
-        responses = (db.session.query(BriefResponse.created_at,
-                                      BriefResponse.id,
-                                      BriefResponse.brief_id,
-                                      BriefResponse.supplier_code,
-                                      Supplier.name.label('supplier_name'))
-                     .join(Supplier)
-                     .filter(BriefResponse.supplier_code == supplier_code,
-                             BriefResponse.brief_id == brief_id,
-                             BriefResponse.withdrawn_at.is_(None))
-                     .all())
+        query = (
+            db.session.query(BriefResponse.created_at,
+                             BriefResponse.id,
+                             BriefResponse.brief_id,
+                             BriefResponse.supplier_code,
+                             Supplier.name.label('supplier_name'),
+                             Supplier.data['contact_name'].label('supplier_contact_name'))
+            .join(Supplier)
+            .filter(
+                BriefResponse.brief_id == brief_id,
+                BriefResponse.withdrawn_at.is_(None)
+            )
+        )
+        if supplier_code:
```

Comment:
any reason you took this out of the `filter` ?

Reply:
it makes the `supplier_code` arg optional, as I needed the brief responses from a buyer's perspective and didn't have a specific supplier code to supply.